### PR TITLE
Update keyring to v2

### PR DESCRIPTION
**Pull request by Powersource**
Based on https://github.com/ssbc/ssb-tribes/pull/75

For https://github.com/ssbc/ssb-tribes/issues/80

### CODE REVIEW

**Comment by Powersource** [comment]
Currently gets stuck at

```
% node test/api/add-new-author-listener.test.js
TAP version 13
# addNewAuthorListener
ok 1 created servers
group ready!
ok 2 admin = returns expected newAuthors
ok 3 admin = returns expected groupId
ssb-tribes/admin ⇒ ssb-tribes/newPerson
ok 4 admin sees newMember being added
ok 5 admin = returns expected groupId
  1 [?]
  2 [?]
  3 [?]
  4 [?]
```

File: index.js [hunk 1]
```diff
@@ -146,13 +149,14 @@ function init (ssb, config) {
...
-      }
-    })
+    console.error('trying to process poBox for msg', m.value.content)
+    //keystore.processPOBox({ poBoxId, poBoxKey }, (err, isNew) => {
+    //  if (err) throw err
+    //  if (isNew) {
+    //    const reason = ['po-box', poBoxId].join()
+    //    rebuildManager.rebuild(reason)
+    //  }
+    //})
```
Comment (Powersource) [inline index.js:159]: what does this do? can i just remove it?

**Comment by Powersource** [comment]
keyring v2 docs https://gitlab.com/ahau/lib/ssb-keyring/-/tree/v2.2.0?ref_type=tags

**Review by mixmix — APPROVED** [review]

**Comment by mixmix** [comment]
Nice clean one, and the tests are all passing :peace_symbol: 
MERGING

**Pull request merged**